CMakeLists.txt

```diff
@@ -44,6 +48,7 @@ if (ENABLE_RHSM_SUPPORT)
 endif ()
 pkg_check_modules (CHECK REQUIRED check)
 pkg_check_modules (REPO REQUIRED librepo)
+pkg_check_modules (LIBMODULEMD REQUIRED modulemd)
```

Comment:
Just call it `MODULEMD` rather than `LIBMODULEMD`. We know it's a library.

Reply:
`MODULEMD` and `LIBMODULEMD` are not same projects.

Reply:
I think it's fine, since `libmodulemd` is the actual pkgconfig module being requested.

Reply:
Should it be like pkg_check_modules (LIBMODULEMD REQUIRED libmodulemd) ?

Reply:
Anyway un both cases it end up with:
Package 'modulemd', required by 'virtual:world', not found
Package 'libmodulemd', required by 'virtual:world', not found

Reply:
I forgot `BuildRequires: libmodulemd-devel` in spec. Fixed.